ui/pages/confirmations/components/confirm/info/approve/approve-details/approve-details.tsx

```diff
@@ -16,7 +16,11 @@ import {
   RecipientRow,
 } from '../../shared/transaction-details/transaction-details';
 
-const Spender = () => {
+const Spender = ({
+  isSetApprovalForAll,
```

Comment:
[nit] we could set a default value to fix type to boolean
```suggestion
  isSetApprovalForAll = false,
```